src/CMakeLists.txt

```diff
@@ -492,6 +492,7 @@ set(libcommon_files
   common/HeartbeatMap.cc
   common/PluginRegistry.cc
   common/ceph_fs.cc
+  common/ceph_posix_fallocate.c
```

Comment:
why not add this implementation to `compat.cc`, and declare the func in `compat.h`?

Reply:
That would be another option, but reason to take this route was that this code was sort of replicated on 2 locations. Where in the second case the `APPLE` version was missing.
And since it is a sort of ceph-wrapper around posix_fallocate I expect it to be more obvious it is stands out on its own.
I'm open for both, please confim if you want to persist with your suggestion.